wezterm-gui/src/overlay/selector.rs

```diff
@@ -155,11 +155,11 @@ impl SelectorState {
                 line.resize(max_width, termwiz::surface::SEQ_ZERO);
             }
             changes.append(&mut line.changes(&attr));
+            changes.push(Change::Text(" \r\n".to_string()));
```

Comment:
I think this should just be the space here, then emit the CRLF after the attributes have changed (where the `" \r\n"` was originally).

It's a subtle difference, but I think moving to the next line before the others are applied might behave slightly wonky in some edge cases.

```suggestion
            changes.push(Change::Text(" ".to_string()));
```